wire-ios/Wire-iOS/Sources/UserInterface/E2EIdentity/SuccessfulCertificateEnrollmentViewController.swift

```diff
@@ -116,6 +125,9 @@ final class SuccessfulCertificateEnrollmentViewController: AuthenticationStepVie
         super.viewDidLoad()
 
         view.backgroundColor = SemanticColors.View.backgroundDefault
+        if isUpdateMode {
+            self.lastE2EIdentityUpdateDate?.storeLastAlertDate(Date.now)
+        }
```

Comment:
This is a side effect nobody expects.

Don't inject any use case into this VC. It's purely for presenting information.
Let the presenter of this view controller invoke the use case instead, please.

Reply:
This gets invoked at many places. That would result in duplicate code everywhere.

Reply:
removed it